### DISCUSSION TITLE
A team of highly skilled and experienced people in search of work for quarkus to raise its impact

### DISCUSSION THREAD

**Discussion by tryptamic** [quarkusio/quarkus#38494] [Jobs and Opportunities]
We are the inventors and implementors of the GenericModel Technology and the hiconic platform which was recently open sourced. While the open sourcing was passionately awaited it was accompanied with the loss of our jobs at the company that previously owned the technology.

We continued to work on the technology anyways and n the last year we made the technology be ready to be publicly used. We used Github's Action, Workflow and Packages features to implement an artifact production pipeline. This completed the open sourcing from being just legal to being productive. During half of this time we got support from former commercial users of the technology but this ran out and we are now freely available on the market.

We would love to be employed as a team that helps to raise quarkus' advantage even more. We think that our technology could be a part of this employment as it would merge the impressive agility and speed of quarkus with our way to work with data and services through a reflected typesystem for normalized models. This would make the speed of quarkus even better at runtime as the highspeed reflection (superior to Java's) allows to process data more effectively on the generic level as it is required by marshallers and other generic algorithms.

I started a discussion in [Design Discussions](https://github.com/quarkusio/quarkus/discussions/38487) in order to get in touch on these matters.

**Comment by maxandersen** [comment]
https://github.com/quarkusio/quarkus/discussions/38492
